docs/app/Examples/elements/Container/Variations/ContainerFluidExample.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable max-len */
+
+import React, { Component } from 'react'
+import { Container, Message } from 'stardust'
+
+export default class ContainerFluidExample extends Component {
+  render() {
+    return (
+      <div>
+        <Message className='info'>
+          Fluid containers are useful for setting text alignment, or other variations on unstyled content
+        </Message>
```

Comment:
Ok, have actioned all of those.  Sorry still getting your code under my 
belt!

> Levi Thomason <mailto:notifications@github.com>
> 24 June 2016 at 17:09
>
> In 
> docs/app/Examples/elements/Container/Variations/ContainerFluidExample.js 
> <https://github.com/TechnologyAdvice/stardust/pull/277#discussion_r68421369>:
>
> > @@ -0,0 +1,22 @@
> > +/* eslint-disable max-len */
> > +
> > +import React, { Component } from 'react'
> > +import { Container, Message } from 'stardust'
> > +
> > +export default class ContainerFluidExample extends Component {
> > + render() {
> > + return (
> > + <div>
> > + <Message className='info'>
> > + Fluid containers are useful for setting text alignment, or other 
> variations on unstyled content
> > + </Message>
>
> Same comment with the |<Message />| here as above. Let's hoist it up 
> out of the example code.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/TechnologyAdvice/stardust/pull/277/files/bd4b8de6b915721a488337a4e1ce7a9194ce895a#r68421369>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AAWHs2B-2HIurEHZ2jwPvh7bkDBeQXRjks5qPAEfgaJpZM4I9hiP>.
>

-- 
Sent from Postbox 
<https://www.postbox-inc.com/?utm_source=email&utm_medium=siglink&utm_campaign=reach>